src/sass/layout/_header.scss

```diff
@@ -0,0 +1,58 @@
+@import 'abstracts/variables';
```

Comment:
Since we are compiling all of our scss to one css file, we actually don't even need to import our variables into individual files. The compilation process will make all of our variables inside of **abstracts/_variables.scss** global and usable in all files.

Reply:
Ah I see. Got it. That's awesome.